test/vol.cpp

```diff
@@ -350,8 +365,45 @@ int main(const int argc, const char** argv)
   }
 
   // If rotate flag is on rotate the polytope
+<<<<<<< HEAD
```

Comment:
this comes from unresolved conflicts when merging. Please resolve. It appears few more times in this branch!